src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -181,9 +181,22 @@ export default class ItemDetailPage extends React.PureComponent<Props, State> {
                       </span>
                       <div className="urn-actions">
                         {isThirdParty(item.urn) ? (
-                          <span className="link" onClick={this.handleEditURN}>
-                            {t('item.edit_urn')}
-                          </span>
+                          <Popup
+                            content={t('collection_item.cannot_edit_urn')}
+                            disabled={false}
+                            position="top center"
+                            trigger={
+                              <div>
+                                <button disabled={item.isPublished} className="link" onClick={this.handleEditURN}>
+                                  {t('item.edit_urn')}
+                                </button>
+                              </div>
+                            }
+                            hideOnScroll={true}
```

Comment:
I'm seeing that the popup does not get removed after moving the mouse out. Is it because of this property?
```suggestion
```

Reply:
No, it was due to the `disabled={false}`  😂 Should be fine without it

Reply:
Well, it works a little bit strange, it you move the button down, it hides it. But if you go up, it still being shown. It's not due to the `hideOnScroll`, I'm not sure why it works like that